### ISSUE TITLE
Missing Aria label for thumbup/down feedback widget. 

### ISSUE THREAD

**Issue by 0xMBL** [ethereum/ethereum-org-website#11725]
### Describe the bug

The Feedback widget button with thumbsup/down doesnt have aria label for screen reader. Once clicked, "Yes", "No", "Close" & is "Is this page helpful?" all have aria labels and can be read on a screen reader but the inital button doesnt. 

I tested using a screen reader & ran lighthouse on the page. Screen reader just said "button" and lighthouse accesibility recommended aria-label.

### To reproduce

1. Go to 'Any page on website'
2. Click on 'N/A'
3. Scroll down to 'Thumbs up/down button'
4. Use screen reader or check code for accesibility.

### Expected behavior

I'd expect the button to be labelled "Feedback" to help with accessibility.

### Would you like to work on this issue?

- [X] Yes
- [ ] No

**Comment by wackerow** [comment]
@0xMBL The FeedbackWidget was just updated to use a Chakra-UI dialog in PR #11906... Checking the DOM, I think the aria-labels may still be missing.

Would you mind confirming this is still an issue? Going to tag this for the community in the meantime

**Comment by 0xMBL** [comment]
Yes, I believe it's still an issue (at least for me)

**Comment by konopkja** [comment]
@wackerow ping
